unit-9/exercise-1.js

```diff
@@ -0,0 +1,17 @@
+const changeSalary = (salary) => {
+	if( salary > 15000000) {
+    const incomeTax = salary * 30 / 100   // incomeTax là tiền thuế thu nhập
+    const netSalary = salary - incomeTax  // netSalary là tiền lương ròng
+    return netSalary
+	}
```

Comment:
Tab gì tào lao thế này
```suggestion
if( salary > 15000000) {
    const incomeTax = salary * 30 / 100   // incomeTax là tiền thuế thu nhập
    const netSalary = salary - incomeTax  // netSalary là tiền lương ròng
    return netSalary
}
```